### PR TITLE
OSDOCS-4473:updates zstream 4.6.62

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-4473](https://issues.redhat.com//browse/OSDOCS-4473): adds zstream update for 4.6.62

Version(s):
4.6

Issue:
[Jira](https://issues.redhat.com/browse/OSDOCS-4473)

Link to docs preview:
[Preview](https://joealdinger.github.io/openshift-docs/OSDOCS-4473/release_notes/ocp-4-6-release-notes.html#ocp-4-6-62)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -3808,4 +3808,23 @@ $ oc adm release info 4.6.61 --pullspecs
 [id="ocp-4-6-61-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-62"]
+=== RHSA-2022:7306 - {product-title} 4.6.62 bug fix update
```
Comment (Wiharris) [inline release_notes/ocp-4-6-release-notes.adoc:3814]:
```suggestion
=== RHSA-2022:7306 - {product-title} 4.6.62 bug fix and security update
```

Comment (Wiharris) [inline release_notes/ocp-4-6-release-notes.adoc:3814]: Interesting, 4.6.62 is showing in the title here, but not rendering in the preview build:

image

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -3808,4 +3808,23 @@ $ oc adm release info 4.6.61 --pullspecs
...
+
+Issued: 2022-11-10
+
+{product-title} release 4.6.61, which includes security fixes, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:7306[RHSA-2022:7306] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7305[RHBA-2022:7305] advisory.
```
Comment (Wiharris) [inline release_notes/ocp-4-6-release-notes.adoc:3818]:
```suggestion
{product-title} release 4.6.62, which includes security fixes, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:7306[RHSA-2022:7306] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7305[RHBA-2022:7305] advisory.
```

**Review by Wiharris — COMMENTED** [review]
/LGTM 

Just a few minor suggestions! :)

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-6-release-notes.adoc:3814]: I pushed an update when I caught the missing 2 and didn't update the preview.

Comment (JoeAldinger) [reply release_notes/ocp-4-6-release-notes.adoc:3814]: My fault @Wiharris this is just a RHBA or bug fix. Appreciate the catch.

**Pull request merged**